### DISCUSSION TITLE
ReadTheDocs versus GH pages

### DISCUSSION THREAD

**Discussion by davidorme** [ImperialCollegeLondon/virtual_rainforest#80] [Q&A]
I was about to start writing an issue to get `vr` up onto ReadTheDocs so that we have a live documentation site, and checking that everything works across local/GH/RTD (see also #49 and #74).

That should _hopefully_ be fairly simple and basically just involve getting a `.readthedocs.yaml` config file set up. There might be some fuss about using `poetry` but there's a good discussion on this (and what happens with Poetry v1.2) on the RTD issues.:

https://github.com/readthedocs/readthedocs.org/issues/4912#issuecomment-1110189914

Before I do that - **do we want to use RTD**? The main argument for that I've always come across is that it supports package versions very cleanly. However people do seem to manage that in GH pages too. 

Any thoughts?

**Comment by dalonsoa** [comment]
Not really. Both are perfectly fine services and I don't have strong feelings either way. I might be more inclined for GH pages simply because it is already here, out of the box, and there's no need to setup a different, external service (with the corresponding accounts, permissions, etc.), but otherwise I've used both in the past with good results.
